### ISSUE TITLE
ViewState.createFromProps is misleading

### ISSUE THREAD

**Issue by a-gagnon** [iTwin/itwinjs-core#6089]
I tried using `ViewState.createFromProps` by passing ViewStateProps that I previously obtained by calling `ViewState.toProps()`. I kept wondering why it always returned undefined until I looked at the implementation which always returns undefined...
It might be a good idea to put a note to use [convertViewStatePropsToViewState](https://www.itwinjs.org/reference/core-frontend/imodelconnection/imodelconnection/imodelconnection.views/#methods) instead and eventually remove this entirely?

* * *

_⚠ Do not edit this section. It is required for imodeljs.github.io ➟ GitHub issue linking_

*   LINK: createFromPropsStatic---core-frontend-link-to-docs
*   TAG: imodeljs-docs-question
*   SOURCE: https://www.itwinjs.org/reference/core-frontend/views/viewstate/createfrompropsstatic/

**Comment by pmconne** [comment]
Yeah, that's weird. It is overridden (statically) in classes like SpatialViewState.
I suspect the author wanted to mark it `abstract`, but TypeScript doesn't support abstract static methods.

The base implementation should probably check the BIS class of the supplied ViewStateProps and invoke the corresponding subclass (e.g., SpatialViewState.createFromProps).
